support-workers/src/main/scala/com/gu/emailservices/GuardianLightEmailFields.scala

```diff
@@ -1,16 +1,46 @@
 package com.gu.emailservices
 
+import com.gu.emailservices.SubscriptionEmailFieldHelpers.formatDate
+import com.gu.support.workers.{
+  ClonedDirectDebitPaymentMethod,
+  CreditCardReferenceTransaction,
+  DirectDebitPaymentMethod,
+  PayPalReferenceTransaction,
+  PaymentMethod,
+  SepaPaymentMethod,
+}
 import com.gu.support.workers.states.SendThankYouEmailState.SendThankYouEmailGuardianLightState
+import org.joda.time.DateTime
+import com.gu.zuora.subscriptionBuilders.GuardianLightSubscriptionBuilder
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class GuardianLightEmailFields {
+class GuardianLightEmailFields(created: DateTime) {
   def build(
       state: SendThankYouEmailGuardianLightState,
   )(implicit ec: ExecutionContext): Future[EmailFields] = {
-    // TODO: populate this when we know what's needed
-    val fields = List()
+    val subscription_details = SubscriptionEmailFieldHelpers
+      .describe(state.paymentSchedule, state.product.billingPeriod, state.product.currency)
+
+    val fields = List(
+      "zuorasubscriberid" -> state.subscriptionNumber,
+      "email_address" -> state.user.primaryEmailAddress,
+      "billing_period" -> state.product.billingPeriod.toString.toLowerCase,
+      "first_payment_date" -> formatDate(
+        created.plusDays(GuardianLightSubscriptionBuilder.gracePeriodInDays).toLocalDate,
```

Comment:
@rupertbates this date logic is also [here](https://github.com/guardian/support-frontend/blob/main/support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/GuardianLightSubscriptionBuilder.scala#L31). I'm sharing the grade period value but I wonder if there's a better way of doing this? Any thoughts?